src/types/index.js

```diff
@@ -21,18 +21,12 @@ const schema = S.object()
   .prop('email', S.string().format('email'))
   .prop(
     'avatar',
-    S.string()
-      .contentEncoding('base64')
-      .contentMediaType('image/png')
+    S.string().contentEncoding('base64').contentMediaType('image/png')
   )
   .required()
   .prop(
     'password',
-    S.string()
-      .default('123456')
-      .minLength(6)
-      .maxLength(12)
-      .pattern('.*')
+    S.string().default('123456').minLength(6).maxLength(12).pattern('.*')
```

Comment:
can you restore the lines changed in this file?

Reply:
This change were made by `npm run format`, but I'll revert it